roles/tarzan/tasks/main.yml

```diff
@@ -16,9 +16,20 @@
               backup=no
   tags: tarzan
 
+- name: add the lua_path to funk_004's environment via the sourceme_upps script
+  lineinfile: dest={{ ngi_pipeline_conf }}/{{ bash_env_upps_script }}
+              line='export LUA_PATH={{ lua_path }}'
+              backup=no
```

Comment:
Yes. Either we consider ngi-pipeline some kind of boss (like you said with the dependency), or we break out the sourceme into a seperate function. Either one works.

I'll merge this for now and we can add that whenever there's time/interest.